Dockerfile

```diff
@@ -19,11 +20,19 @@ RUN apk update && apk add \
     && tar -xzf "/tmp/v${BATS_VERSION}.tar.gz" -C /tmp/ \
     && bash "/tmp/bats-${BATS_VERSION}/install.sh" /usr/local \
     \
+    && wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && tar -C /usr/local/bin -xzvf dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && rm dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    \
     && rm -rf /var/cache/apk/* && rm -rf /tmp/*
 
+
 ## Configure Service
 
-COPY install/main.dist.cf /etc/postfix/main.cf
+ENV CERTIFICATE_PUBLIC=/etc/postfix/cert/smtp.cert
+ENV CERTIFICATE_PRIVATE=/etc/postfix/cert/smtp.key
+
+COPY install/main.dist.cf.tmpl /etc/postfix/main.cf.tmpl
```

Comment:
I'd like to overwrite this file(`main.cf`) every time if `CERTIFICATE_PUBLIC` is set because that's the purpose of why we set the ENV variable.

Reply:
By default the content of the expanded main.cf will be "overwritten" because the location of /etc/postfix is not mounted as a volume, hence it will be lost between restarts.

But _if_ the user mounts the `/etc/postfix/main.cf` I don't think is desirable to overwrite it. Is more about not blocking eventual needed tweaks in that file.